### ISSUE TITLE
static support without dpd-static-support

### ISSUE THREAD

**Issue by jadeiX** [GibbsConsulting/django-plotly-dash#174]
Hello, 
Thank you for creating and maintaining this awesome package. I have all my static files including bootstrap 4 in my Django static project folder.  I am wondering if it is possible to direct django-plotly-dash to use these files instead of installing dpd-static-support? (which seems to only contain the same bootstrap files)
I would appreciate the help.

**Comment by GibbsConsulting** [comment]
The purpose of the `dpd-static-support` package is to provide mappings of the needed files to ones provided locally alongside other Django static files.

Is your goal to use a different mapping? If so then you can either change the mapping of this package - pull requests always most welcome! - or use something else to provide the mapping.

**Comment by jadeiX** [comment]
`dpd-static-support ` has bootstrap 4.1.3 and I use a local copy of bootstrap 4.3.1. which is already in my static files folder. Thus far what I have done is install the package, run collectstatic and manually change the files in the package to bootstrap 4.3.1 
then I also change the file mapping.py file to:

```
static_substitutions = [
    ('https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css','dpd_static_support/css'),
    ('https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js','dpd_static_support/js'),
    ("https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.15.0/umd", 'dpd_static_support/js'),
    ("https://code.jquery.com/jquery-3.4.1.min.js", "dpd_static_support/js/jquery.min.js"),

]

substitutions = [

    ('integrity="sha384-ggOyR0iXCbMQv3Xipma34MD+dH/1fQ784/j6cY/iJTQUOhcWr7x9JvoRxT2MZw1T"', ''),
    ('integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"', ''),
    ('integrity="sha256-CSXorXvZcTkaix6Yvo6HppcZGetbYMGWSFlBw8HfCJo="', ''),
    ('integrity="sha256-7BfFV/dSvQT4pGBvRAIt6JDXsehb92DQqpGUndLCPQ4="', ''),

]
```
Probably this is not the right way to go about this. I tried to look in the in the installation files but couldnt figure out what part of the script creates the mapping.py file.
I apologies for the multiple questions, but how do I change the mapping? 
In other words, how can I tell `django-plotly-dash` to look into a different static folder ( ie where i keep   bootstrap 4.3.1 files)

**Comment by GibbsConsulting** [comment]
The short-term fix is to just add (rather than replace) the extra entries you've listed above. Ideally a better way can be found, perhaps by configuring the bootstrap file locations directly.

Right now, to look in a different folder, just change the path in the mapping file for each of the files/directories.

**Comment by jadeiX** [comment]
Thank you for the advice. Quick question, what script in `dpd-static-support ` creates the mapping file?

**Comment by GibbsConsulting** [comment]
There isn't one at the moment; existing entries have been created by hand.
